examples/gocean/eg2/Makefile

```diff
@@ -103,11 +103,11 @@ single_prof.exe: ${INF_LIB} ${PROFILE_LIB} psy_prof.f90 kernels psy_prof.o alg_p
 ${PROFILE_LIB} ${NVTX_LINK} -o single.exe
 
 psy.f90 alg_gen.f90: alg.f90
-	${PSYCLONE} -l -api "gocean1.0" -s ./acc_transform.py -opsy psy.f90 \
+	${PSYCLONE} -l all -api "gocean1.0" -s ./acc_transform.py -opsy psy.f90 \
```

Comment:
What is the purpose of -l all in this example? There is no reference to its use in the README. Is it required? If so, its use should be mentioned somewhere (in the README's?). If not then it should be removed.

Reply:
I've no idea. It appears complete unnecessary so I've removed it.